test-jest/cartesian/ReferenceDot.spec.tsx

```diff
@@ -74,17 +73,17 @@ describe('<ReferenceDot />', () => {
         <ReferenceDot x="201106" y={20} stroke="#666" label="201106" alwaysShow />
       </BarChart>,
     );
-    expect(wrapper.find('.recharts-reference-dot-dot').length).to.equal(1);
-    expect(wrapper.find('.recharts-label').length).to.equal(1);
+    expect(container.querySelectorAll('.recharts-reference-dot-dot')).toHaveLength(1);
+    expect(container.querySelectorAll('.recharts-label')).toHaveLength(1);
   });
 
-  it('Render custom lable when label is set to be a react element', () => {
-    const Label = ({ text, ...props }) => (
+  test('Render custom lable when label is set to be a react element', () => {
+    const Label = ({ text, ...props }: { text: any }) => (
```

Comment:
Please do not use any if possible.

Reply:
I think it's ok in test code.

Reply:
`any` is still used here - `text` should be a string at the very least - fine for the sake of the migration but lets try to be strict with our typing